packages/api/tests/endpoints/test-rules.js

```diff
@@ -449,22 +666,96 @@ test('PUT returns 400 for name mismatch between params and payload',
     t.falsy(record);
   });
 
+test('PUT replaces onetime rule for rerun', async (t) => {
+  const expectedRule = {
+    ...omit(testRule, ['queueUrl', 'provider', 'collection']),
+    state: 'ENABLED',
+  };
+  expectedRule.action = 'rerun';
+
+  await request(app)
+    .put(`/rules/${testRule.name}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .send(expectedRule)
+    .expect(200);
+
+  const { body: actualRule } = await request(app)
+    .get(`/rules/${testRule.name}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  t.truthy(actualRule);
```

Comment:
```suggestion
  const expectedRule = {
    ...omit(testRule, ['provider', 'collection']),
    action: 'rerun',
  };

  const { body: actualRule } = await request(app)
    .put(`/rules/${expectedRule.name}`)
    .set('Accept', 'application/json')
    .set('Authorization', `Bearer ${jwtAuthToken}`)
    .send(expectedRule)
    .expect(200);

  t.deepEqual(actualRule, testRule);
```

All we need to prove is that what comes back from the PUT request itself is the original rule with no alterations

Reply:
[`717d0e5` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/717d0e50dc816dd503b437d92a583a563a6c0766)